cdk/lib/__snapshots__/recipes-backend.test.ts.snap

```diff
@@ -113,26 +113,23 @@ import urllib.request
 s3 = boto3.client("s3")
 
 EVENTBRIDGE_CONFIGURATION = 'EventBridgeConfiguration'
-
 CONFIGURATION_TYPES = ["TopicConfigurations", "QueueConfigurations", "LambdaFunctionConfigurations"]
 
 def handler(event: dict, context):
   response_status = "SUCCESS"
   error_message = ""
   try:
     props = event["ResourceProperties"]
-    bucket = props["BucketName"]
     notification_configuration = props["NotificationConfiguration"]
-    request_type = event["RequestType"]
     managed = props.get('Managed', 'true').lower() == 'true'
+    skipDestinationValidation = props.get('SkipDestinationValidation', 'false').lower() == 'true'
```

Comment:
Is this a change to keep something imposed by CDK happy? Or perhaps it's not important because this is just for the test?

Reply:
Yes, it's in the snapshot; it's an artefact of one of the library bumps. Bit rubbish to get lumps of Python code in there but it's out of our control 🤷